docs/components/modeler/desktop-modeler/lint-rules/lint-rules.md

```diff
@@ -0,0 +1,82 @@
+---
+id: lint-rules
+title: Lint rules
+description: "Lint rule plugins allow you to add your own lint rules to the Desktop Modeler."
+---
+
+[bpmnlint](https://github.com/bpmn-io/bpmnlint) plugins allow you to add custom lint rules for BPMN diagrams or disable existing rules.
+
+## Getting started
+
+To get started, you can use [this example project](https://github.com/camunda/camunda-modeler-custom-linter-rules-plugin).
+
+Take the following steps:
+
+1. Clone the repository:
```

Comment:
If we want to encourage people to share their work, I'd rather suggest to fork the repo instead.